src/signals/incident/components/form/DescriptionInput/DescriptionInput.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import TextArea from 'components/TextArea';
+import DescriptionInfo from '../DescriptionInfo';
+
+function get(e, meta, parent) {
```

Comment:
Any particular reason this is not an anonymous function?

Reply:
Old code. I changed that.